rebar.lock

```diff
@@ -0,0 +1 @@
+[].
```

Comment:
Wat?

Reply:
No dependencies = empty lock file. I find it strange that project_plugins are not locked, but so it is.
I'd prefer to leave it in on the principle that rebar.lock should not be added to .gitignore.

Reply:
Ok